hs_core/hydroshare/utils.py

```diff
@@ -844,105 +831,65 @@ def resource_file_add_process(resource, files, user, extract_metadata=False,
+                         move=False):
     """
     Add a ResourceFile to a Resource.  Adds the 'format' metadata element to the resource.
     :param resource: Resource to which file should be added
     :param f: File-like object to add to a resource
-    :param folder: name of the folder path to which the file needs be uploaded
-    :param fed_res_file_name_or_path: the logical file name of the resource content file for
-                                      federated iRODS resource or the federated zone name;
-                                      By default, it is empty. A non-empty value indicates
-                                      the file needs to be added into the federated zone, either
-                                      from local disk where f holds the uploaded file from local
-                                      disk, or from the federated zone directly where f is empty
-                                      but fed_res_file_name_or_path has the whole data object
-                                      iRODS path in the federated zone
-    :param fed_copy_or_move: indicate whether the file should be copied or moved from private user
-                             account to proxy user account in federated zone; A value of 'copy'
-                             indicates copy is needed, a value of 'move' indicates no copy, but
-                             the file will be moved from private user account to proxy user account.
-                             The default value is None, which indicates N/A, or not applicable,
-                             since the files do not come from a federated zone, and this copy or
-                             move operation is not applicable, but any value other
-                             than 'copy' or 'move' is regarded as N/A.
+    :param source_name: the logical file name of the resource content file for
+                        federated iRODS resource or the federated zone name;
+                        By default, it is empty. A non-empty value indicates
+                        the file needs to be added into the federated zone, either
+                        from local disk where f holds the uploaded file from local
+                        disk, or from the federated zone directly where f is empty
+                        but source_name has the whole data object
+                        iRODS path in the federated zone
+    :param move: indicate whether the file should be copied or moved from private user
+                 account to proxy user account in federated zone; A value of False
+                 indicates copy is needed, a value of True indicates no copy, but
+                 the file will be moved from private user account to proxy user account.
+                 The default value is False.
 
     :return: The identifier of the ResourceFile added.
     """
     if f:
+        # does not seem to be needed
```

Comment:
It's in the code now and I don't know why. 
Would prefer to leave it in as documentation in case things break. 
Then I'll know what to try........